src/Essentials/src/Essentials.csproj

```diff
@@ -39,6 +39,7 @@
     <Compile Include="**\*.windows.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
     <Compile Include="**\*.windows.*.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
     <PackageReference Include="Microsoft.Web.WebView2" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" />
```

Comment:
Please stop referencing the meta package. With 1.8 you should only need to reference Microsoft.WindowsAppSDK.WinUI. The meta package is really only there for compatibility with components that are built with 1.7